src/cli_utils.rs

```diff
@@ -337,14 +337,16 @@ impl<W: Write> Printer<W> {
 
                 if self.list_files {
                     self.print_subrow()?;
-
+                    let mut reports: Vec<_> = language.reports.clone();
```

Comment:
Why clone this?

Reply:
Because language is borrowed and ```sort_by``` function requires mutable access

Reply:
You could create a new `Vec<&R>` from `language.reports` and sort that one. You can use `.map` and `.collect`.

Reply:
I don't know if this is how I should do it, if I try like this to get a Vec<Report> , it says that the Copy Trait is not implemented. 

![image](https://github.com/UPB-CS-OpenSourceUpstream/tokei/assets/119881489/e51f2fac-3401-4641-8c6c-d424b1838173)

And I can't use a Vec<&Report> because I would have to change the code below to accept &Report
`let (a, b): (Vec<_>, Vec<_>) = reports
                            .iter()
                            .partition(|r| r.stats.blobs.is_empty());`

Reply:
Please use 

```rust
post code here
```

to post code instead of pictures.

Reply:
You want to get a `Vec<&Report>`, instead you are trying to take ownership here.

Reply:
Pushed a commit now with the requested change. I will run ```fmt``` on it after feedback,